### ISSUE TITLE
XGBoost constraints don't work with verbose mode

### ISSUE THREAD

**Issue by pobonomo** [Gurobi/gurobi-machinelearning#225]
To reproduce run the Janos/XGboost notebook and set `verbose=True` in the call to `add_predictor_constr`, you get the error:
```
            estimators.append(
>               TreeEstimator(
                    self.gp_model,
                    tree,
                    self.input,
                    tree_vars[:, i, :],
                    self.epsilon,
                    timer,
                    self.verbose,
                    **kwargs,
                )
            )
E           TypeError: __init__() got multiple values for argument 'verbose'
```

**Comment by pobonomo** [comment]
Normally every modeling object has a `verbose` attribute and the base class deals with initializing it. I don't remember why I have a `_verbose` attribute here. It's probably an error unless I was trying to achieving something very strange.

**Comment by pobonomo** [comment]
Fixed with release 1.3.3
